### DISCUSSION TITLE
Cannot find rolling-release and stable versions

### DISCUSSION THREAD

**Discussion by Hanner72** [AzuraCast/AzuraCast#6400] [Q&A] [answered]
Hello,
I am using Azuracast in "Rolling Release" and would like to switch to "Stable". However, the rolling release version must be older than or at least the same as the stable version. Where can I find out the two versions?
Either I'm too blind or too stupid.
Please show me the way.

**Comment by Piratenkanon** [comment]
Run:
./docker.sh install
again and change to Stable

**Comment by LordHelmchen666** [comment, marked as answer]
You can find documentation here https://docs.azuracast.com/en/getting-started/updates/release-channels

Stable Release Dates you can finde here https://github.com/AzuraCast/AzuraCast/blob/main/CHANGELOG.md

Your actual Azuracast Version can be found at the bottom of your Azuracast Dashboard which looks like `Powered by AzuraCast • Rolling Release #996bc43 (2023-06-26 10:42) • Docker • PHP 8.2`

**Reply by Hanner72** [reply]
Thanks!
